src/container/srv_target.c

```diff
@@ -104,6 +118,9 @@ cont_child_free_ref(struct daos_llink *llink)
 {
 	struct ds_cont_child *cont = cont_child_obj(llink);
 
+	cont->sc_vos_aggregating = 0;
+	ABT_thread_yield();
```

Comment:
On server shutdown (see dss_srv_handler()), we wait for all ULTs exit then do TLS fini for all modules where the container cache is cleared, won't it cause deadlock (aggregate ULT won't exit before clear container cache)?